lib/resources/majors.tsx

```diff
@@ -4365,6 +4356,382 @@ const minorAMS_New: Minor = {
+            '<b>Physics I</b> <br />' +
+            'Select one of the following: <br />' +
+            'AS.171.101 General Physics: Physical Science Majors I <br />' +
+            'AS.171.103 General Physics: Biological Science Majors I <br />' +
+            'AS.171.107 General Physics for Physical Sciences Majors (AL)',
+          required_credits: 4,
+          criteria: 'AS.171.101[C]^OR^AS.171.103[C]^OR^AS.171.107[C]',
+        },
+        {
+          description:
+            '<b>Physics Laboratory I</b> <br /> AS.173.111 General Physics Laboratory I',
+          required_credits: 1,
+          criteria: 'AS.173.111[C]',
+        },
+        {
+          description:
+            '<b>Physics II</b> <br />' +
+            'Select one of the following: <br />' +
+            'AS.171.102 General Physics: Physical Science Majors II <br />' +
+            'AS.171.104 General Physics/Biology Majors II <br />' +
+            'AS.171.108 General Physics for Physical Science Majors (AL)',
+          required_credits: 4,
+          criteria: 'AS.171.102[C]^OR^AS.171.104[C]^OR^AS.171.108[C]',
+        },
+        {
+          description:
+            '<b>Physics Laboratory II</b> <br /> AS.173.112 General Physics Laboratory II',
+          required_credits: 1,
+          criteria: 'AS.173.112[C]',
+        },
+      ],
+    },
+    {
+      name: 'Biology Sequence',
+      required_credits: 6,
+      min_credits_per_course: 3,
+      pathing: 1,
+      description:
+        'Select one biology course with lab of the following. Exam credit not applicable. For more detail please visit ' +
+        'https://krieger.jhu.edu/neuroscience/bs-program/requirements/',
+      criteria:
+        'AS.020.151[C]^OR^AS.020.153[C]^OR^AS.020.152[C]^OR^AS.020.154[C]^OR^' +
+        'AS.020.305[C]^OR^AS.020.315[C]^OR^AS.250.253[C]^OR^AS.250.254[C]^OR^' +
+        'AS.020.303[C]^OR^AS.020.340[C]^OR^AS.020.306[C]^OR^AS.020.316[C]^OR^' +
+        'AS.020.374[C]^OR^AS.020.377[C]',
+      double_count: [
+        'Humanities (H) Distribution',
+        'Social Sciences (S) Distribution',
+        'Other (N/E/Q) Distribution',
+        'Writing Intensive',
+      ],
+      fine_requirements: [
+        {
+          required_credits: 4,
+          description:
+            '<b>General Biology I with lab</b> <br /> AS.020.151 General Biology I<br /> AS.020.153 General Biology Laboratory I',
+          criteria: 'AS.020.151[C]^OR^AS.020.153[C]',
+        },
+        {
+          required_credits: 4,
+          description:
+            '<b>General Biology II with lab</b> <br /> AS.020.152 General Biology II<br /> AS.020.154 General Biology Lab II',
+          criteria: 'AS.020.152[C]^OR^AS.020.154[C]',
+        },
+        {
+          required_credits: 5,
+          description:
+            '<b>Genetics with lab</b> <br /> AS.020.303 Genetics<br /> AS.020.340 Developmental Genetics Lab',
+          criteria: 'AS.020.303[C]^OR^AS.020.340[C]',
+        },
```

Comment:
For this path, you can even choose Protein Engineering and Biochemistry Lab or Biochemistry project lab. Also change required credits to 4